db/database.go

```diff
@@ -94,7 +98,7 @@ type DatabaseContext struct {
 	State              uint32                   // The runtime state of the DB from a service perspective
 	ExitChanges        chan struct{}            // Active _changes feeds on the DB will close when this channel is closed
 	OIDCProviders      auth.OIDCProviderMap     // OIDC clients
-	PurgeInterval      int                      // Metadata purge interval, in hours
+	PurgeInterval      time.Duration            // Metadata purge interval, in hours
```

Comment:
```suggestion
	PurgeInterval      time.Duration            // Metadata purge interval
```

Reply:
This comment has been revised in latest commit.